applications/StructuralMechanicsApplication/custom_elements/total_lagrangian.cpp

```diff
@@ -520,8 +520,7 @@ void TotalLagrangian::CalculateSensitivityMatrix(
     KRATOS_TRY;
 
     const auto& r_geom = GetGeometry();
-    if (rDesignVariable == SHAPE_SENSITIVITY)
-    {
+    if (rDesignVariable == SHAPE_SENSITIVITY) {
```

Comment:
i see completely no reason to change this formatting...

Reply:
I did it just because the rest of the file is using this style...